service/src/scripts/profile-setup.ts

```diff
@@ -4,25 +4,26 @@ import { getWeb3Interface } from "./web3-interface";
 import { validateEnvVars } from "../utils/validateEnv";
 import { exit } from "process";
 import { FILE_READ_EXCEPTION, RPC_EXCEPTION } from "../utils/exit-codes";
-import { archLogger } from "../utils/chalk-theme";
+import { archLogger } from "../logger/chalk-theme";
 import { BigNumber, ethers } from "ethers";
 import { requestApproval } from "./approve_utils";
 
 import jsonfile from "jsonfile";
 import { getOnchainProfile, inMemoryStore } from "../utils/onchain-data";
+import { logProfile } from "../cli/utils";
 
 validateEnvVars();
 
-export enum ProfileArgNames {
-  DIGGING_FEE = "digging-fee",
-  REWRAP_INTERVAL = "rewrap-interval",
-  FREE_BOND = "free-bond",
+export enum ProfileOptionNames {
+  DIGGING_FEE = "diggingFee",
+  REWRAP_INTERVAL = "rewrapInterval",
+  FREE_BOND = "freeBond",
 }
 
 export interface ProfileParams {
-  diggingFee: BigNumber;
-  rewrapInterval: number;
-  freeBond: BigNumber;
+  diggingFee?: BigNumber;
+  rewrapInterval?: number;
+  freeBond?: BigNumber;
```

Comment:
when using update, any of these params may not be provided (and in which case the already-existing profile value will be used).